tests/test_state_cycle.py

```
@@ -69,6 +69,39 @@ def test_edit_nonexistent(self, patch_rx_session):
+        """Editing a cycle to use another cycle's name should be rejected."""
+        make_cycle(name="Sprint 1")
+        cycle_b = make_cycle(name="Sprint 2")
+        state = _make_state(cycle_id_value=str(cycle_b.id))
+        result = state.save_cycle_edits(cycle_b.id, {"name": "Sprint 1", "threshold": "80"})
+        assert result != 0  # toast error
+        session = patch_rx_session
+        session.expire_all()
+        updated = session.exec(select(CycleModel).where(CycleModel.id == cycle_b.id)).first()
+        assert updated.name == "Sprint 2"
+
+    def test_edit_cycle_same_name_allowed(self, patch_rx_session, make_cycle):
```

Ruff (0.15.6)

[warning] Unused method argument: `patch_rx_session`

(ARG002)